src/picterra/base_client.py

```diff
@@ -24,6 +24,31 @@
 CHUNK_SIZE_BYTES = 8192  # 8 KiB
 
 
+def _get_distr_name():  # only for test_headers_user_agent_version__fallback
```

Comment:
Can we just slightly expand this comment to explain that this is to "inject an non-existing package name to test the fallback behavior of `_get_ua` in tests" ?

(If I understand the intent correctly)